Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -85,6 +89,8 @@ public SeedData Cross(SeedData a, SeedData b)
         CrossBool(ref result.Sentient, a.Sentient);
         CrossBool(ref result.Ligneous, a.Ligneous);
         CrossBool(ref result.Bioluminescent, a.Bioluminescent);
+        CrossBool(ref result.TurnIntoKudzu, a.TurnIntoKudzu);
+        CrossBool(ref result.Screaming, a.Screaming);
```

Comment:
`mandrogaSound` is not crossed; did you mean to implement that?

Reply:
this sound is not randomly generated upon mutation and every plants have 1 sound for screaming .
if there will be multiple sounds for screaming, i thinck it would be better to mention them in yml.